content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -32,7 +32,7 @@ You now need to ensure that the kubectl completion script gets sourced in all yo
 
 {{< tabs name="kubectl_bash_autocompletion" >}}
 {{< tab name="User" codelang="bash" >}}
-echo 'source <(kubectl completion bash)' >>~/.bashrc
+echo "source $(kubectl completion bash)" >>~/.bashrc
```

Comment:
I don't think this changes require , `kubectl` completion bash  work fines for me by using command 
`echo 'source <(kubectl completion bash)' >>~/.bashrc`